beacon-chain/core/blocks/exit_test.go

```diff
@@ -135,8 +135,6 @@ func TestProcessVoluntaryExits_AppliesCorrectStatus(t *testing.T) {
 }
 
 func TestVerifyExitAndSignature(t *testing.T) {
-	undo := util.HackDenebMaxuint(t)
-	defer undo()
```

Comment:
This was renamed to HackElectraMaxuint and serves the same purpose. It's no longer needed in this particular test.